test/integration_test.py

```diff
@@ -641,6 +630,56 @@ def worker(_):
                     expect_execution = fetch == first_fetch
                     self.assertEqual(1 if expect_execution else 0, len(execution_ids))
 
+    def _manifest_filters(self, catalog: CatalogName) -> JSON:
+        # IT catalogs with just one public source are always indexed
```

Comment:
The longest line in this paragraph wraps before column 79,
I also ran the PyCharm formater and there were no additional changes to this file.
Otherwise, I'm unsure as to what other formatting enhancements I may do.

Reply:
^^ I was wrong, this indeed needed reformating.